src/Tools/AnalyzerRunner/Program.cs

```diff
@@ -65,9 +79,15 @@ public static async Task Main(string[] args)
                 ProfileOptimization.SetProfileRoot(options.ProfileRoot);
             }
 
-            Stopwatch stopwatch = Stopwatch.StartNew();
+            var stopwatch = PerformanceTracker.StartNew();
             var properties = new Dictionary<string, string>
             {
+#if NETCOREAPP
+                // This property ensures that XAML files will be compiled in the current AppDomain
+                // rather than a separate one. Any tasks isolated in AppDomains or tasks that create
+                // AppDomains will likely not work due to https://github.com/Microsoft/MSBuildLocator/issues/16.
+                { "AlwaysCompileMarkupFilesInSeparateDomain", bool.FalseString },
```

Comment:
Should this be moved to MSBuildWorkspace?

Reply:
I'm not ready to make that proposal yet.